packages/react-aria-components/test/RadioGroup.test.js

```diff
@@ -452,4 +452,16 @@ describe('RadioGroup', () => {
     expect(group).not.toHaveAttribute('aria-describedby');
     expect(group).not.toHaveAttribute('data-invalid');
   });
+
+  it('should call onBlur when focus leaves the radio', async() => {
+    let onBlur = jest.fn();
+    let {getAllByRole} = renderGroup({onBlur});
+    let radio = getAllByRole('radio')[0];
+
+    act(() => radio.focus());
```

Comment:
it's best to use user actions when possible
```suggestion
    await user.tab();
```

and lets check that focus/blur isn't called when arrow keying through radios
```
    await user.keyboard('[ArrowRight]');
    // expect focus/blur not to have been called more
```

Reply:
Fixed by the following commit.

[refac: user user action instead of act](https://github.com/adobe/react-spectrum/pull/5441/commits/b2746d599e730aa288fb30d1525a92238cecfa43)